slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/model/Metadata.java

```diff
@@ -31,6 +31,10 @@ public Metadata(
     this.genesisValidatorsRoot = genesisValidatorsRoot;
   }
 
+  public int getFormatVersion() {
+    return formatVersion;
+  }
+
   @JsonGetter(value = "interchange_format_version")
   public String getFormatVersionAsString() {
```

Comment:
Why do we need this? Looks like it used in tests but these test could just use getFormatVersion instead?

Reply:
format version is written to JSON as a string, not as a numeral - so need to make sure we use this when serialising json